src/troubleshooting/general/magento-2.4.0-checkout-error-selecting-local-payments.md

```diff
@@ -12,31 +14,33 @@ This article talks about a solution for known issue in Magento during checkout w
 
 ## Issue
 
-Preconditions
+### Preconditions
```

Comment:
```suggestion
<ins>Preconditions</ins>:
```

Reply:
Reject : <ins> is not used in Markdown. It appears nowhere in this repo or even DevDocs

Reply:
Actually, because of our unique setup, and style guide/format, you -must- use the "ins" html tag in order for the underline function to be shown in ZenDesk, which is where this article is published.

All Support KB articles are published in ZenDesk. This is different than what DevDocs and MerchDocs use.

I am fully aware it is not Markdown-approved, but it is the -only way- the end result can be underlined in ZD. I welcome suggestions, and you can iterate underline ways if you want to try to find a Markdown way, but no one on our end has found another reliable way to underline to fit our style guide/format.

Reply:
Okay thats perfectly acceptable! Also I have discussed naming nomenclature on the Slack Channel and am now happy with the decision. I just wanted to make sure it was correct.